conf/deployment.yaml

```diff
@@ -56,6 +56,36 @@ rules:
   verbs:
     - list
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tf-artifacts
```

Comment:
if we can't get this from tf-api, we can generalize the name of the tf-api service account and use it for the artifacts pod as well..